FileReaderApp/ViewController.m

```diff
@@ -82,7 +86,20 @@ - (void)matchExtension:(NSString *)clickedFileType {
         [FileAlert showAlertWithController:self message:@"Selected File Type Mismatch"];
```

Comment:
@ahammad-reyad  localizable string file not needed in this project
@Tahia50 but can be moved to a constant file